py/build.desc

```diff
@@ -89,3 +89,11 @@ py_env(
   ],
   dest = "build/python"
 )
+
+py_install(
+	name = "install"
+)
```

Comment:
Please use two space indent, as this is actually just Ruby code.  Alternatively put it on one line.